interceptor_ext_test.go

```diff
@@ -234,7 +234,6 @@ func (h *headerInterceptor) WrapUnary(next connect.UnaryFunc) connect.UnaryFunc
 		h.inspectResponseHeader(req.Spec(), res.Header())
 		return res, nil
 	}
-	return connect.UnaryFunc(call)
```

Comment:
There were two places in this test that were unnecessarily wrapped in `connect.UnaryFunc(...)`.